solver/pb/ops.proto

```diff
@@ -93,8 +93,13 @@ message OpMetadata {
 	// Description can be used for keeping any text fields that builder doesn't parse
 	map<string, string> description = 2;  
 	WorkerConstraint worker_constraint = 3;
+	ExportCache export_cache = 4;
 }
```

Comment:
I think we should replicate the comment above to here, but it can be later. LGTM